pilot/pkg/model/push_context.go

```diff
@@ -621,6 +629,31 @@ func sortServicesByCreationTime(services []*Service) []*Service {
 	return services
 }
 
+// Caches list of service accounts in the registry
+func (ps *PushContext) initServiceAccounts(env *Environment) error {
+	for _, svcs := range ps.privateServicesByNamespace {
```

Comment:
we have an allServices variable. You could use that.

Reply:
you mean `env.Services()`?